admin/javascript/src/silverstripe-backend.js

```diff
@@ -45,8 +45,15 @@ class SilverStripeBackend {
    * @return object - Promise
    */
   post(url, data) {
-    return this.fetch(url, { method: 'post', credentials: 'same-origin', body: data })
-      .then(checkStatus);
+    return this.fetch(url, {
+      method: 'post',
+      headers: new Headers({
+        'Content-Type': 'application/x-www-form-urlencoded',
```

Comment:
Hm, I think hardcoding a header here isn't ideal. How about `Object.assign({}, headers, {'Content-Type': 'application/x-www-form-urlencoded'})` with `headers` passed in as an optional method argument? That should also apply to the other HTTP verbs in the backend here.